### ISSUE TITLE
Prevent the same project from displaying twice on impact area landing page

### ISSUE THREAD

**Issue by TylerRadford** [hotosm/hotosm-website#256]
If a project is displayed as a highlighted project, could we avoid showing it again in the section below "All Impact Area x Projects"

![image](https://user-images.githubusercontent.com/9853903/40921114-93b81d04-67dc-11e8-88f9-ce9f84eac0a4.png)

**Comment by brendangatens** [comment]
I can fix this. I already did so on country pages but didn't follow through on Impact Area pages.

The same thing is happening on the landing page, with a difference cause though. It looks like projects started to show in both _Highlighted projects_ and _All projects_ once we implemented sorting the latter by date: https://github.com/hotosm/hotosm-website/commit/f544c44a8ddac4b77b3b6964617e1228d445b288

If we want to order the _All projects_ list by date, we'll have to capture the three highlighted projects (the first three when sorted by position in Siteleaf) and exclude them somehow. I'll look at this as well.
